### DISCUSSION TITLE
Maintaining an API count

### DISCUSSION THREAD

**Discussion by valentinedwv** [dagster-io/dagster#18242] [Q&A]
I have a web service that has a daily rate limit, and i would like to reset it to zero on a daily basis.

Can I store that in a resource somehow so it's shared across calls, or do I make it an asset.

**Comment by benpankow** [comment]
Hi David,

If your intent is to track the number of calls to the webservice over the course of a day, you'll likely need to use some sort of external data store which allows this count to be shared between runs. You could use a resource to track this count, but that resource would need to read and write from the external store - generally, runs happen in different Python instances & potentially even different containers if deployed to something like Docker or Kubernetes. You can see this discussion which mentions a couple libraries that might help with this goal: https://github.com/dagster-io/dagster/discussions/5878
